third_party/BUILD

```diff
@@ -605,6 +605,39 @@ filegroup(
     ],
 )
 
+pkg_tar(
+    name = "cpp_util",
```

Comment:
nit: Can you please use a more meaningful name?

Reply:
The sole purpose of the target is to package all the files `src/main/cpp/util`. I hope the comments make everything clearer. Let me know if you have any suggestions.